implementation.md

```diff
@@ -1554,3 +1554,272 @@ In the `Video` and `Audio` objects, sellers must only include one form of floor
+                    "id": "fac13741-0648-436a-88cf-aceafdf45c9a",
+		    "atype": "3",
+                }
+            ]
+        },
+        {
+            "inserter": "sspdomain.com",
+            "source": "dspdomain.com”,
+            "mm": 3,
+            "uids": [
+                {
+                    "id": "3d89748d-74bb-44f1-9908-298090dc2944",
+		    "atype": "1",
+                }
+            ]
+        }
+    ]
+}
+}
+```
+note that <code>mm</code>:3 is retained in this example, because the match is built on top of an ID that was itself based on cross-domain authentication (e.g. via hashed email)
+
+#### 7.12.4.3 Derrived IDs Across Multiple Properties or Devices
+##### Example 5: SSP matches their own ID to an email-based hash to provide a universal ID to the DSP in the buyeruid field (based on prior arrangement to put it in buyeruid)
+```
+{
+"user": {
+    "buyeruid": "fac13741-0648-436a-88cf-aceafdf45c9a",
+    "eids": [
+        {
+            "source": "sspdomain.com”,
+            "mm": 1,
+            "uids": [
+                {
+                    "id": "3d89748d-74bb-44f1-9908-298090dc2944",
+		    "atype": "1",
+                }
+            ]
+        },
+        {
+            "matcher": "idvendor.com",
+	    "inserter": "sspdomain.com",
+            "source": "universalid.com”,
+            "mm": 4,
+            "uids": [
+                {
+                    "id": "fac13741-0648-436a-88cf-aceafdf45c9a",
+		    "atype": "3",
+                }
+            ]
+        }
+    ]
+}
+}
+```
+##### Example 6: On a CTV request, the publisher also provides the IFA and an alternative ID of an Apple device within that same household, based on authenticated logins across apps
+
+```
+{
+"user": {
+    "eids": [
+        {
+	    "matcher": "app-pub.com",
+	    "inserter": "app-pub.com",
+	    "source": "apple.com”,
+            "mm": 3,
+            "uids": [
+                {
+                    "id": "fac13741-0648-436a-88cf-aceafdf45c9a",
+		    "atype": "2",
+                }
+            ]
+        },
+        {
+            "matcher": "app-pub.com",
+	    "inserter": "app-pub.com",
+            "source": "alt-id.org”,
+            "mm": 3,
+            "uids": [
+                {
+                    "id": "e7f8c5f2-6b4e-4565-8b0a-53d8b730143e",
+		    "atype": "3",
+                }
+            ]
+        }
+    ]
+}
+}
+“device”: {
+    "ifa": "df3a42f4-a388-43e9-a691-b0d308c963b4",
+}
+```
+
+##### Example 7: On a CTV request, the publisher provides the IFA of the TV, but also adds the IFA of an Apple device within that same household to the extended IDs array, based on the household’s IP and a device graph
+```
+{
+"user": {
+    "eids": [
+        {
+	    "matcher": "graphvendor.com",
+	    "inserter": "app-pub.com",
+	    "source": "apple.com”,
+            "mm": 4,
+            "uids": [
+                {
+                    "id": "fac13741-0648-436a-88cf-aceafdf45c9a",
+		    "atype": "3",
```

Comment:
shouldn't this be `"atype": 2` rather than 3? An Apple IFA is a device ID, even if the match-method is based on inference.

Reply:
i had it as 3 because its x-device so it'd be a person based identifier

Reply:
I don't think that's the spirit of an "agent type". From AdCOM: "This list identifies the user agent types a user identifier is from." An IFA from an Apple device is a device identifier, not a user identifier. This is why I made the callout above that it's still a device ID, even if it was matched in based on inference that it's the same user at that IP. It has not become a RampID or something -- it's still an IFA.